_maps/RandomRuins/LavaRuins/lavaland_surface_lust.dmm

```diff
@@ -0,0 +1,60 @@
+"a" = (/turf/template_noop,/area/template_noop)
```

Comment:
None of these have air do they?

Reply:
If they do have are, what air truffs is it pulling?

Reply:
They should take the turf and area of whatever tile it WOULD have replaced